dev/openchpl_data-model.sql

```diff
@@ -2877,6 +2877,24 @@ CREATE TABLE IF NOT EXISTS openchpl.user_certification_body_map (
 		MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
 );
 
+CREATE TABLE IF NOT EXISTS openchpl.user_test_lab_map (
```

Comment:
The trigger is using a table called user_testing_lab_map but this is using user_test_lab_map